components/breadcrumb/BreadcrumbItem.razor

```diff
@@ -37,20 +37,22 @@
 
 @code {
 
-	[Parameter]
-	public RenderFragment ChildContent { get; set; }
+    [Parameter]
+    public RenderFragment ChildContent { get; set; }
 
-	[Parameter]
-	public RenderFragment Overlay { get; set; }
+    [Parameter]
+    public RenderFragment Overlay { get; set; }
 
-	[Parameter]
-	public string Href { get; set; }
+    [Parameter]
+    public string Href { get; set; }
 
-	[CascadingParameter]
-	public Breadcrumb BreadCrumb { get; set; }
+    [Parameter]
+    public EventCallback<MouseEventArgs> OnClick { get; set; }
```

Comment:
I think it's better to pass in the current `BreadcrumbOption`.

Reply:
Should we pass back the `BreadcrumbItem` itself? From what I could tell, the `BreadcrumbOption` class is not used anywhere yet. I just didn't remove it because it was public.

Reply:
Oh, that fine, but it's better to pass some info of current item to user

Reply:
Yeah, makes sense. I went ahead and did a `Tuple<MouseEventArgs, BreadcrumbItem>` so the user has access to the click arguments and the item since both have useful information. How does that look?

Reply:
That's fine. Do you plan to fix HTML differences(#2658) here?

Reply:
I was thinking of doing that in a separate PR after these changes get in. Would that just be PR'd to master or somewhere else since it will completely change the HTML for existing users? Wasn't sure if that would be considered a breaking change or not.

Reply:
I think this is a breaking change for CSS, not for HTML or Blazor. CSS has been updated in the latest version, but HTML hasn't caught up.

Reply:
It was a pretty small change to update the HTML markup so I went ahead and did that too. It is now in this PR.